Update guidance regarding the private key format and encoding (#445)
logstash-plugins/logstash-input-beats: docs/index.asciidoc

  * There is no default value for this setting.

SSL key to use.

> NOTE: This key needs to be in the PKCS8 format and PEM encoded. This conversion can be done using the https://www.openssl.org/docs/man1.1.1/man1/openssl-pkcs8.html[openssl pkcs8] 
> command. For example, the command to convert a PEM encoded PKCS1 private key to a PEM encoded, non-encrypted PKCS8 key is:

Reviewer: ```suggestion
This key must be in the PKCS8 format and PEM encoded. 
You can use the https://www.openssl.org/docs/man1.1.1/man1/openssl-pkcs8.html[openssl pkcs8] command to complete the conversion.
For example, the command to convert a PEM encoded PKCS1 private key to a PEM encoded, non-encrypted PKCS8 key is:
```

> ["source","sh",subs="attributes"]

Reviewer: The way you have it isn't incorrect. I use the simplest version that works to avoid propaging complexity.
